styles/blocks/f-button-group.css

```diff
@@ -58,27 +36,29 @@ I think it would look awesome!
 			<button class="f-button-group__button_underline">Who we are</button>
 			<button class="f-button-group__button_underline">Dank memes</button>
 		</div>
-	</section>
-
-<!-- this should be used on the website -->
-</div>
 ```
-
-
 */
 
 @import "../variables";
 
-.f-button-group { 
+.f-button-group {}
 
+.f-button-group__button {
+	display: inline;
+	
+	width: 100%;
 }
 
 .f-button-group__button::first-letter {
```

Comment:
@troldmand This selector is perfect but doesn't work because all of the BEM **E**lements only have **M**odifiers. See my previous comment